documentation/eng_sys.md

```diff
@@ -0,0 +1,50 @@
+# Engineering System Checks
+
+* [Build and Test](#build-and-test)
+* [Analyze Stages](#analyze-stages)
+
+## Build and Test
+
+Our build system runs PR changes against the latest two versions of Go on both Windows and Linux.
+
+## Analyze
+
+### Link Verification Check
+Verifies all of the links are valid in your README files. This step also checks that locale codes in links are removed. If this is failing first check if you have locale codes (ie. `en-us`) in your links, then check to see if the link works locally.
+
+If you are trying to add a link that will exist in the next PR (ie. you are adding a samples README or migration guide), you can use an `aka.ms` link or use a temporary link (ie: `https://microsoft.com`) and create a follow-up PR to correct the temporary link.
+
+### Lint
+Linting will not fail a CI check, but it will show possible issues with your design. Some of the most common are:
```

Comment:
Is this true?  I thought CI requires lint-free.

Reply:
Updated